README.md

```diff
@@ -16,7 +16,7 @@ Tesseract OCR is used for the text recognition itself.
 
 - Tesseract OCR is installed
   - for Windows see https://github.com/UB-Mannheim/tesseract/wiki
-  - for Linux, Mac see https://github.com/tesseract-ocr/tesseract/wiki
+  - for Linux, Mac see https://github.com/tesseract-ocr/tessdoc/blob/master/Installation.md
```

Comment:
```suggestion
  - for Linux, Mac see https://tesseract-ocr.github.io/tessdoc/Installation.html
```

Reply:
Thanks -- I missed that this existed as a page